clusterloader2/docs/oom_detection.md

```diff
@@ -10,6 +10,9 @@ Given that `ClusterOOMsTracker` is based on Kubernetes events and events are
 best effort by their nature, the reported summary is not guaranteed to
 accurately describe what really happened - some of the OOMs may be missed.
 
+For Kubemark clusters where nodes are hollow and `node-problem-detector` is
```

Comment:
In Kubemark tests clusters nodes are hollow ... , so this will only track OOMs occuring in master components.

Reply:
Done.